src/libs/actions/Policy.js

```diff
@@ -344,6 +345,39 @@ function createPolicyExpenseChats(policyID, invitedEmailsToAccountIDs, betas) {
     return workspaceMembersChats;
 }
 
+/**
+ * Build personal details objects for a given list of logins and accountIDs
+ *
+ * @param {Array<string>} logins Array of user logins
+ * @param {Array<number>} accountIDs Array of user accountIDs
+ * @returns {Object} - object with optimisticData, successData and failureData (object of personal details objects)
+ */
+function buildPolicyPersonalDetails(logins, accountIDs) {
```

Comment:
The function name is not the most accurate, this data is not explicit to policies. Can we move this function to somewhere more fitting? e.g. `PersonalDetails.js`. Also I think we should indicate that the function will only return onyx data for non existing users.

Reply:
Can we move that to `PersonalDetails` instead? Having `personalDetails` as an object is faster to access.

Reply:
Maybe we should add another value to `PersonalDetailsUtils`? keeping `personalDetails` as array and adding `allPersonalDetails` as object.

Then we make make a follow up PR to rewrite the util functions that use the array to use the object and remove the array variable.